### DISCUSSION TITLE
Ticket viewing issue

### DISCUSSION THREAD

**Discussion by darryl20** [Peppermint-Lab/peppermint#362] [General]
![image](https://github.com/user-attachments/assets/8e606af7-82a8-4f02-9d76-aa87d19a9454)
has anyone come across this error while viewing tickets after the ticket is created using the ticket portal link?

**Comment by Chiwinks** [comment]
I get the same error. I notice that you have the same version as I do (0.4.9). I'm confused as to why `pepperlabs/peppermint:latest` pulls 0.4.9. [https://github.com/Peppermint-Lab/peppermint/releases](https://github.com/Peppermint-Lab/peppermint/releases) mentions the latest release being 0.4.8. [https://peppermint.sh](https://peppermint.sh) says the latest version is 0.4.5. I wonder if 0.4.9 is a pre-release and somehow made it into the production release.

**Reply by Chiwinks** [reply]
I'm also curious as to why historical versions are not made available for just such issues.

**Comment by Chiwinks** [comment]
Check out [https://github.com/Peppermint-Lab/peppermint/discussions/351](https://github.com/Peppermint-Lab/peppermint/discussions/351)

**Comment by potts99** [comment]
This is now fixed :) sorry for the delay in this @darryl20 @Chiwinks
